integration_tests/e2e/conditions/create/createConditions.cy.ts

```diff
@@ -30,15 +30,95 @@ describe('Create Conditions', () => {
     Page.verifyOnPage(SelectConditionsPage)
   })
 
-  it('should create a prisoners Conditions, triggering validation on every screen', () => {
+  it('should create a prisoners self-declared Conditions, triggering validation on every screen given user does not have permission to create diagnosed conditions', () => {
```

Comment:
The diff has gone a bit odd here - it looks like a new test, but its the existing test, but with a slightly different title, and the use of the signIn stub with specific roles, and the assertion of the page heading on L45
Other than than it's exactly the same test